team-manager/src/test/resources/application-test.yml

```diff
@@ -0,0 +1,9 @@
+#spring:
```

Comment:
delete

Reply:
this is for my next pr. I intially wanted to make intergration tests here, but its already too big pr without them